### PR TITLE
Add missing dependency

### PR DESCRIPTION
**Pull request by jspricke**

### CODE REVIEW

File: stdeb.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [DEFAULT]
-Depends: subversion, mercurial, git-core, bzr, python-yaml, python-vcstools (>= 0.1.38), python-catkin-pkg, python-rosdistro (>=0.3.0), python-wstool (>=0.1.12)
-Depends3: subversion, mercurial, git-core, bzr, python3-yaml, python3-vcstools (>= 0.1.38), python3-catkin-pkg, python3-rosdistro (>=0.3.0), python3-wstool (>=0.1.12)
+Depends: subversion, mercurial, git-core, bzr, python-yaml, python-vcstools (>= 0.1.38), python-catkin-pkg, python-rosdistro (>=0.3.0), python-wstool (>=0.1.12), rospkg
```
Comment (wjwwood) [inline stdeb.cfg]: I believe the dependency is called `python-rospkg`:

http://repositories.ros.org/ubuntu/main/pool/main/p/python-rospkg/

File: stdeb.cfg [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+Depends3: subversion, mercurial, git-core, bzr, python3-yaml, python3-vcstools (>= 0.1.38), python3-catkin-pkg, python3-rosdistro (>=0.3.0), python3-wstool (>=0.1.12), rospkg
```
Comment (wjwwood) [inline stdeb.cfg]: And `python3-rospkg`:

http://repositories.ros.org/ubuntu/main/pool/main/p/python3-rospkg/

**Comment by wjwwood** [comment]
@jspricke I also removed unsupported Python versions (should fix CI).

**Comment by jspricke** [comment]
Done.

**Comment by jspricke** [comment]
@wjwwood anything I need to change here?

**Comment by wjwwood** [comment]
No the changes look good, but it needs to be rebased. I'll try to do that, but I might have to close this and open a new one to do it.

**Comment by jspricke** [comment]
I can do it.

**Comment by wjwwood** [comment]
Oops, I just did it.

**Comment by wjwwood** [comment]
Ok, yours overwrote mine, but that's fine. Rebased one way or the other.

**Comment by jspricke** [comment]
:D

**Pull request merged**